spec/api/v1/merchants_controller_spec.rb

```diff
@@ -23,4 +23,26 @@
 
     assert_equal merchant.name, response_merchant["name"]
   end
+
+  scenario "finds items associated with one merchant" do
+    merchant = merchants(:one)
+
+    get "/api/v1/merchants/#{merchant.id}/items"
+    assert_response :success
+    parsed_data = JSON.parse(response.body)
+
+    expect(parsed_data.count).to eq(2)
```

Comment:
we'll probably want to add some assertions about the content of the data; we should be getting item data back